README.md

```diff
@@ -1 +1,70 @@
-# To-Do-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TO-DO LIST
+TO-DO LIST is a basic website that allows users to add/remove task from a to-do list. 
+
+# To Do list: list structure
+- Learning objectives
+- Use webpack to bundle JavaScript.
+- Learn how to use proper ES6 syntax.
+- Use ES6 modules to write modular JavaScript.
+
+## Screenshot
+
+![Screenshot](https://user-images.githubusercontent.com/88809610/154808929-c35b5e17-57a9-4757-a232-d576ae57b01a.png)
+
+
+## Built With
+
+- HTML, CSS, JavaScript, GIT,
+- Technologies used: GitHub, VScode, nodejs, webpack, lodash.
+
+## Live Demo
+
+[Live Demo Link](https://forhemer.github.io/To-Do-List/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A device that support running browser applications.
+- access to internet connection.
+
+
+### Setup
+
+#### View pages from the browser
+
+- If you just want to check the webpage/webapp, you have it available on the live demo link.
+
+#### Install locally the code of the application
+
+In order to install a local version of this project and please do the following steps:
+- Install the latest version of any code editor.
+- install the latest version of GIT.
+- Go to the [repository](https://github.com/ForHemer/To-Do-List) and clone it.
```

Comment:
- [ ] It would be nice to include information about how to clone, install and run the server locally. 👍

Reply:
I can see that this has been resolved 👍🏽